app/models/concerns/episode_ad_breaks.rb

```diff
@@ -0,0 +1,25 @@
+require "active_support/concern"
+
+module EpisodeAdBreaks
+  extend ActiveSupport::Concern
+
+  included do
+    alias_error_messages :ad_breaks, :segment_count
```

Comment:
Threw all the "ad_breaks" stuff into a concern.  To avoid polluting `episode.rb` ... and in case we decide to remove this at some point.  (I'm a bit skeptical about this field... I worry we're just going to confuse all our existing users).

Reply:
Hopefully we'll get some fast feedback on this soon!